src/Components/Operations.js

```diff
@@ -0,0 +1,48 @@
+import React, {Component} from 'react'
+import { Link } from 'react-router-dom'
+import DelayLink from 'react-delay-link'
+
+class Operations extends Component{
+    constructor(){
+        super()
+        this.state = {
+            amount: "",
+            vendor: "",
+            category: ""
+        }
+    }
+    // handleDeposit = () => {
+    //     this.props.addFunc({amount: this.state.amount * 1, vendor: this.state.vendor, category: this.state.category})
+    // }
+
+    // handleWithdraw = () => {
+    //     this.props.addFunc({amount: this.state.amount * -1, vendor: this.state.vendor, category: this.state.category})
+    // }
+
+    handleOperation = (num) => {
```

Comment:
can use object destructuring:
`const { amount, category, vendor } = this.state`
`this.props.addFunc({ amount: amount*num, category, vendor })`